app/imports/ui/pages/ListCompanies.jsx

```diff
@@ -33,8 +34,16 @@ class ListCompanies extends React.Component {
   renderPage() {
     const { companies } = this.props;
     return (
-        <Container>
+        <Container style={{ marginTop: '80px' }}>
           <Grid>
+
+
+            <Grid.Row>
```

Comment:
@mattkim009  remove skills from the listcompanies page

Reply:
done